x-pack/plugins/alerting/server/authorization/alerting_authorization_kuery.ts

```diff
@@ -78,3 +71,77 @@ export function ensureFieldIsSafeForQuery(field: string, value: string): boolean
   }
   return true;
 }
+
+export const buildRuleTypeFilter = (
+  ruleTypes: Set<RegistryAlertTypeWithAuth>,
+  opts: AlertingAuthorizationFilterOpts,
+  alertSpaceId?: string
+): JsonObject => {
+  const allFilters = Array.from(ruleTypes).map(({ id, authorizedConsumers }) => {
+    const ruleIdFilter = {
+      bool: {
+        should: [
+          {
+            match: {
+              [opts.fieldNames.ruleTypeId]: id,
+            },
+          },
+        ],
+        minimum_should_match: 1,
+      },
+    };
+    const spaceIdFilter =
+      alertSpaceId != null && opts.fieldNames.spaceIds != null
+        ? {
+            bool: {
+              filter: [{ term: { [opts.fieldNames.spaceIds]: alertSpaceId } }],
+            },
+          }
```

Comment:
I think this can just be

```suggestion
        ? { term: { [opts.fieldNames.spaceIds]: alertSpaceId } }
```

Reply:
Updated, thanks!